build/dependencies.yaml

```diff
@@ -107,9 +107,8 @@ dependencies:
     version: 1.19
     refPaths:
     - path: build/build-image/cross/VERSION
-    # TODO(liggitt): Uncomment once images are updated to go1.19
-    #- path: hack/lib/golang.sh
-    #  match: minimum_go_version=go([0-9]+\.[0-9]+)
```

Comment:
revert this change, per https://github.com/kubernetes/enhancements/tree/master/keps/sig-release/3744-stay-on-supported-go-versions#proposal we'll continue to run unit and integration tests on the original go version for 1.23, which requires leaving the minimum go version in golang.sh as 1.17.x

can drop the TODO or change the comment to explain why this is 1.17

Reply:
done, thanks